src/sa_web/static/js/views/app-view.js

```diff
@@ -279,13 +283,23 @@ var Shareabouts = Shareabouts || {};
     },
     loadLandmarks: function() {
       var self = this;
-      _.each(_.values(this.options.landmarkConfigs), function(landmarkConfig) {
-        if (landmarkConfig.placeType) {
-          self.landmarkCollections[landmarkConfig.id].fetch({
-            attributesToAdd: { location_type: landmarkConfig.placeType }
+
+      // REFACTOR
+      // Here's where I'm adding the datasetSlug value for landmark places
```

Comment:
What this code is doing is adding the datasetSlug into our landmark models when we fetch them from the API. Why are we doing it? It doesn't seem necessary, and I don't see why we need to access the datasetSlug from the model.